code/modules/admin/verbs/adminpm.dm

```diff
@@ -223,9 +223,10 @@
 	var/compliant_msg = trim(lowertext(msg))
 	var/irc_tagged = "[sender](IRC)"
 	var/list/splits = splittext(compliant_msg, " ")
+	var/usage_string = "Usage: ticket <close|resolve|icissue|reject|reopen \[ticket #\]|list>"

```

Comment:
technically it's static already as the string table will look it up and find it.
but making it actually ```/static``` avoids a string table lookup so, still a +